src/pyorcidator/import_info.py

```diff
@@ -1,16 +1,32 @@
 from .helper import *
 from urllib.parse import quote
 import click
+from pathlib import Path
 
 
 @click.command(name="import")
-@click.option("--orcid", prompt=True, help="The ORCID Id to look up")
-def main(orcid: str):
-    qs = render_orcid_qs(orcid)
+@click.option(
+    "--orcids",
+    prompt="ORCID ID or a path to list of ORCID IDs",
```

Comment:
That's actually a great point, @cthoyt. I've read the same advice before but it still didn't cross my mind. My main worry in this case is to avoid unnecessary code repetition, but I'll refactor it as soon as I can. Thank you!